### ISSUE TITLE
Clase Player y PlayerInfo [E2]

### ISSUE THREAD

**Issue by vicente999** [RodrigoToroIcarte/IIC2113-2023-1#151]
Hola! Hice una clase Player con los siguientes atributos:
<img width="260" alt="image" src="https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/53841116/26db103b-041b-4890-9c6d-6683896b4a85">
Asi guardo toda la info del jugador, sin embargo ya esta la clase "PlayerInfo". Lo que me gustaría hacer es poder combinarlos con herencia, pero tengo entendido que no podemos editar el archivo PlayerInfo.cs y no puedo hacer public class Player:PlayerInfo

Que puedo hacer? O si me recomiendan hacer otra cosa

**Comment by RodrigoToroIcarte** [comment]
Exacto. No puedes editar el código del `RawDealView`. De todas formas no creo que sea buena idea heredar de `PlayerInfo`. El propósito del objeto `PlayerInfo` es simplemente pasar los datos del jugador a la vista para que los pueda mostrar. Mi consejo es que agregues al `Player` un método del estilo `GetPlayerInfo()` que retorne un `new PlayerInfo(...)` con la información actual del `Player`. Así cuando quieras mostrar la información de la vista puedes llamar a ese método para obtener el `PlayerInfo` de cada jugador y dárselos a la vista.

**Comment by vicente999** [comment]
Perfecto! Asi es como lo tenia actualmente. Muchas gracias!
